src/Files.App/Controllers/SidebarPinnedController.cs

```diff
@@ -43,17 +43,20 @@ public async Task InitializeAsync()
 		public async Task ReloadAsync()
 		{
 			await LoadAsync();
-			Model.RemoveStaleSidebarItems();
+			Model?.RemoveStaleSidebarItems();
 		}
 
 		private async Task LoadAsync()
 		{
 			StorageFolder Folder = await FilesystemTasks.Wrap(() => ApplicationData.Current.LocalFolder.CreateFolderAsync("settings", CreationCollisionOption.OpenIfExists).AsTask());
 			if (Folder is null)
 			{
-				Model.AddDefaultItems();
-				await Model.AddAllItemsToSidebar();
-				return;
+				Model?.AddDefaultItems();
+                        	if (Model is not null)
+                        	{
+                    			await Model.AddAllItemsToSidebar();
+                        	}
```

Comment:
I think you should move `Model?.AddDefaultItems()` inside `if`. That way we can get rid of `?` and we're not double checking the variable